homeassistant/components/media_player/__init__.py

```diff
@@ -123,7 +123,8 @@
 })
 
 MEDIA_PLAYER_MUTE_VOLUME_SCHEMA = MEDIA_PLAYER_SCHEMA.extend({
-    vol.Required(ATTR_MEDIA_VOLUME_MUTED): vol.Coerce(bool),
+    # pylint: disable=no-value-for-parameter
+    vol.Required(ATTR_MEDIA_VOLUME_MUTED): vol.Boolean(),
```

Comment:
We could also add `boolean = vol.Boolean()` to `helpers.config_validation` and probably avoid the pylint annotations.